mbid_mapping/mapping/release_colors.py

```diff
@@ -259,11 +262,14 @@ def incremental_update_release_color_table():
                        , release.gid AS release_mbid
                        , mime_type
                        , date_uploaded
+                       , year
                     FROM cover_art_archive.cover_art caa
                     JOIN cover_art_archive.cover_art_type cat
                       ON cat.id = caa.id
-                    JOIN musicbrainz.release
+                    JOIN musicbrainz.release r
                       ON caa.release = release.id
+                    JOIN release_first_release_date rfrd
```

Comment:
This will exclude releases that have a cover art but no release year. If you don't intend to do that, use a LEFT JOIN.

Reply:
I just saw the SQL and I had misread it initially yeah you need to use a LEFT JOIN here to avoid missing albums.

Reply:
Good catch, thanks!